pkg/imagepolicy/kyverno/engine.go

```diff
@@ -0,0 +1,91 @@
+package kyverno
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	runtimeClient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+	"github.com/sealerio/sealer/common"
+	imagecommon "github.com/sealerio/sealer/pkg/define/options"
+	"github.com/sealerio/sealer/pkg/imageengine"
+	"github.com/sealerio/sealer/pkg/infradriver"
+	k "github.com/sealerio/sealer/pkg/runtime/kubernetes"
+	apimachineryRuntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+type Engine struct {
+}
+
+func NewKyvernoImagePolicyEngine() (*Engine, error) {
+	return &Engine{}, nil
+}
+
+func (engine *Engine) IsImagePolicyApp(appName string) (bool, error) {
+	return appName == "kyverno", nil
```

Comment:
add TODO here

we should make it configurable later

Reply:
how about add a prefix for default image policy plugin name, just like `sealerio-image-policy-kyverno`